CSharp/DevVmPowershell/Helpers/RequestModels/InstanceSettingManagerCreateRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Helpers.RequestModels
+{
+	public class InstanceSettingManagerCreateRequest
+	{
+		public instanceSetting instanceSetting { get; set; }
+	}
+
+	public class instanceSetting
```

Comment:
@calimetikcura This is because the InstanceSettingManagerUpdateRequest also has a class called InstanceSetting so I'm not able to have two classes that are the exact same casing in the same location

Reply:
ok. it makes sense now.
I think there might be a better way to handle it than like this. but not a road blocker. I am going mark this as resolved.